umake/frameworks/ide.py

```diff
@@ -338,6 +338,22 @@ def __init__(self, category):
                          packages_requirements=['openjdk-7-jdk', 'jayatana'],
                          icon_filename='webide.png')
 
+class CLion(BaseJetBrains):
+    """The JetBrains CLion IDE"""
+    download_page_url = "https://www.jetbrains.com/clion/download/download_thanks.jsp?os=linux"
+    executable = "clion.sh"
+
+    def __init__(self, category):
+        super().__init__(name="CLion",
+                         description=_("CLion"),
```

Comment:
I used this description to be consistent with the other JetBrains IDEs in this file, but I can change it to something more descriptive if you'd like.